### ISSUE TITLE
String vs. Boolean for $selinux_enabled 

### ISSUE THREAD

**Issue by stefanlasiewski** [puppetlabs/puppetlabs-docker#144]
The description for $selinux_enabled implies that it's a boolean:

```
# [*selinux_enabled*]
#   Enable selinux support. Default is false. SELinux does  not  presently
#   support  the  BTRFS storage driver.
#   Valid values: true, false
```

However, the code for selinux_enabled seems to be used as a string.

I found at least two sections which treat this boolean as a string:

https://github.com/puppetlabs/puppetlabs-docker/blob/43dcd50ef3dd536668f59fc253c6888404288a10/manifests/init.pp#L399

```
  Optional[String] $selinux_enabled                         = $docker::params::selinux_enabled,
```

And https://github.com/puppetlabs/puppetlabs-docker/blob/43dcd50ef3dd536668f59fc253c6888404288a10/manifests/init.pp#L488-L492

```
  if $selinux_enabled {
    assert_type(Pattern[/^(true|false)$/], $selinux_enabled) |$a, $b| {
      fail translate(('selinux_enabled must be true or false'))
    }
  }
```

**Comment by sheenaajay** [comment]
@stefanlasiewski Thanks for the detailed information.  PR is ready https://github.com/puppetlabs/puppetlabs-docker/pull/145

**Comment by sheenaajay** [comment]
@stefanlasiewski in init.pp used the data type Boolean for selinux_enabled and in params.pp used undef so it will continue to work in the same way.

**Comment by davejrt** [comment]
Fixed in PR #145
